elastic/kibana
# [Fleet] Tagging of integration assets

## Summary

Solves https://github.com/elastic/kibana/issues/123904

Added logic to create Managed and integration name tags and reference them with installed package assets.

To verify:
- Add integration with assets e.g. System, Apache
- Go to `Tags` view and verify that the tags are created and linked to the assets of each package. The `Managed` tag should appear once only. 
- On uninstalling and adding the integration again, the integration name tag should not be duplicated.

<img width="701" alt="image" src="https://user-images.githubusercontent.com/90178898/181040540-fdff6188-56c6-4c2f-9477-11cdfd172a9a.png">
<img width="693" alt="image" src="https://user-images.githubusercontent.com/90178898/181041250-46cf8553-dade-4d07-b6c1-0a0eefb5b98e.png">
<img width="912" alt="image" src="https://user-images.githubusercontent.com/90178898/181041334-f490f647-7c63-4303-84ae-4b6474e0e24e.png">

Open questions / issues:
- Should we add a description to the tags?
- The white color of tags look good on dark mode, though not the best on light mode, should we change it?

<img width="691" alt="image" src="https://user-images.githubusercontent.com/90178898/181041543-c40d28d0-84de-4663-b5a5-23efc7bdf50a.png">

Pending:
- Unit and functional tests

### Checklist
